### ISSUE TITLE
Assignment 2 Q 1.3 

### ISSUE THREAD

**Issue by almas2019** [STAT545-UBC/Discussion#553]
I am a bit confused regarding the instructions for this question:
Filter gapminder to all entries that have experienced a **drop** in life expectancy. Be sure to include a new variable that’s the **increase** in life expectancy in your tibble. Hint: you might find the lag() or diff() functions useful.
This seems a bit contradictory.

Does this mean filter out countries that have a drop in life expectancies over all the years or only keep those have experienced a drop?
Is it the whole gapminder dataset and not the filtered one from 1.1? Also are we looking at the general trend over all the years , or just the 1970s?
Thank you in advance.

**Comment by armetcal** [comment]
I was wondering something similar. I know we're supposed to use the gapminder dataset, but I'm specifically wondering whether we're supposed to analyze:

- Any lines in the data that have a reduced life expectancy from the previous line
- Any countries that have experienced a drop in life expectancy overall
- Any rows where the country has experienced a drop in life expectancy from year to year (AKA doesn't count lines where we're changing countries)

**Comment by almas2019** [comment]
@vincenzocoia

**Comment by wvictor14** [comment]
@almas2019 ,
I agree it sounds contradictory. But the instructions does not say that the new variable "an increase in life expectancy" **can't** have negative values. I think you should also try using the whole gapminder dataset (not the filtered one from 1.1).

@armetcal My interpretation is the second or third one (leaning towards the third one), maybe @vincenzocoia can comment.

Hopefully that helps..

**Comment by vincenzocoia** [comment]
Yep, you can just filter "increase in lifeExp" to be less than 0 to get those rows with a drop in lifeExp.

**Comment by armetcal** [comment]
Great, thanks!
